steve/webedit.py

```diff
@@ -31,10 +31,12 @@
 HOST = 'localhost'
 PORT = 8000
 
+import os
+
 
 def get_data(cfg, fn):
-    data_files = load_json_files(cfg)
-    data_files = [d for d in data_files if d[0][5:] == fn]
+    data_files = load_json_files(cfg, path=True)
+    data_files = [d for d in data_files if d[0].rsplit(os.sep, 1)[-1] == fn]
```

Comment:
I'd be surprised if we're in a situation where that optimization matters, but it's not a big deal either way.